src/writeData/subscriptions/components/SubscriptionDetails.tsx

```diff
@@ -70,56 +81,61 @@ const SubscriptionDetails: FC<Props> = ({
               edit={edit}
             />
           </Overlay.Body>
-          <Overlay.Footer>
-            <Button
-              text="Close"
-              color={ComponentColor.Tertiary}
-              onClick={() => {
-                event('close button clicked', {}, {feature: 'subscriptions'})
-                history.push(`/orgs/${org.id}/${LOAD_DATA}/${SUBSCRIPTIONS}`)
-              }}
-              titleText="Cancel update and return to Subscriptions list"
-              type={ButtonType.Button}
-              testID="update-subscription-form--cancel"
-            />
-            <Button
-              text="Edit"
-              color={edit ? ComponentColor.Success : ComponentColor.Secondary}
-              onClick={() => {
-                event('edit button clicked', {}, {feature: 'subscriptions'})
-                setEdit(!edit)
-              }}
-              type={ButtonType.Button}
-              titleText="Edit"
-              testID="update-subscription-form--edit"
-            />
-            <Button
-              text="Next"
-              color={ComponentColor.Secondary}
-              onClick={() => {
-                event('next button clicked', {}, {feature: 'subscriptions'})
-                setFormActive('parsing')
-              }}
-              type={ButtonType.Button}
-              titleText="Next"
-              testID="update-subscription-form--submit"
-            />
-            <Button
-              text="View Data"
-              color={ComponentColor.Success}
-              onClick={() => {
-                event(
-                  'view data button clicked',
-                  {},
-                  {feature: 'subscriptions'}
-                )
-                history.push(`/orgs/${org.id}/notebooks`)
-              }}
-              type={ButtonType.Button}
-              testID="update-subscription-form--view-data"
-              status={ComponentStatus.Default}
-            />
-          </Overlay.Footer>
+
+          {!singlePage ? (
+            <Overlay.Footer>
+              <Button
+                text="Close"
+                color={ComponentColor.Tertiary}
+                onClick={() => {
+                  event('close button clicked', {}, {feature: 'subscriptions'})
+                  history.push(`/orgs/${org.id}/${LOAD_DATA}/${SUBSCRIPTIONS}`)
+                }}
+                titleText="Cancel update and return to Subscriptions list"
```

Comment:
I think this footer could be put into a component, maybe? It looks like the code is duplicated in each of the pages right now and is wrapped with the `{!singlePage ? ...}` flag

Reply:
yeah i could make it a single component where the parsing info for what to navigate to next is handed in

Reply:
ok i've pulled it out, ended up being more faffy as the parsing form has different buttons as well.